index.bs

```diff
@@ -757,23 +755,24 @@ dictionary XRReferenceSpaceOptions {
 };
 </pre>
 
-An {{XRReferenceSpace}} is obtained by calling {{XRSession/requestReferenceSpace()}}, which creates an instance of an interface extending {{XRReferenceSpace}}, determined by the {{XRReferenceSpaceOptions/type}} value of the {{XRReferenceSpaceOptions}} dictionary passed into the call:
+An {{XRReferenceSpace}} is obtained by calling {{XRSession/requestReferenceSpace()}}, which creates an instance of an {{XRReferenceSpace}} or an interface extending it, determined by the {{XRReferenceSpaceOptions/type}} value of the {{XRReferenceSpaceOptions}} dictionary passed into the call. The {{XRReferenceSpaceOptions/type}} indicates the tracking behavior that the reference space will exhibit:
+
+  - Passing a {{XRReferenceSpaceOptions/type}} of <dfn enum-value for="XRReferenceSpaceType">identity</dfn> creates an {{XRReferenceSpace}} instance. It represents a reference space without any tracking behavior. The {{XRPose/transform}} of any {{XRPose}} queried from an {{identity}} reference space MUST be an [=identity transform=] offset by the {{originOffset}}.
```

Comment:
Given the discussion in https://github.com/immersive-web/webxr/issues/565 this characterization seems inaccurate? The pose queried changes based on the base space used, a more accurate characterization is that "getViewerPose" should always return the identity transform (potentially offset with originOffset).

Reply:
Reworked this text, though I think it's going to need to be updated again when we land some of the planed changes re: describing these spaces in terms of the native origin as you've started looking at.